src/Windows/Avalonia.Win32/OleDataObjectHelper.cs

```diff
@@ -363,6 +362,20 @@ public unsafe static uint WriteDataToHGlobal(IDataTransfer dataTransfer, DataFor
             }
         }
 
+        if (ClipboardFormatRegistry.PngSystemDataFormat.Equals(format)
+            || ClipboardFormatRegistry.PngDataFormat.Equals(format))
+        {
+            var bitmap = dataTransfer.TryGetValue(DataFormat.Bitmap);
+            if (bitmap != null)
+            {
+                using var stream = new MemoryStream();
+                bitmap.Save(stream);
+
+                return WriteBytesToHGlobal(ref hGlobal, stream.ToArray().AsSpan());
```

Comment:
we could probably do here without a copy, there is MemoryStream.GetBuffer(), but it might be larger than you need

Reply:
Thats probably over-optimization. We are rarely copying images that often to have issues with memory copies. Best to be stable in this case.